app/about/us/page.tsx

```diff
@@ -0,0 +1,137 @@
+    href: "/ashley-lee",
+    image: "/images/people/ashley_main.jpg",
+  },
+  {
+    name: "Ashley Lee",
+    title: "Senior Research Software Engineer",
+    href: "/ashley-lee",
+    image: "/images/people/ashley_main.jpg",
+  },
+  {
+    name: "Ashley Lee",
+    title: "Senior Research Software Engineer",
+    href: "/ashley-lee",
+    image: "/images/people/ashley_main.jpg",
+  },
+  {
+    name: "Ashley Lee",
+    title: "Senior Research Software Engineer",
+    href: "/ashley-lee",
+    image: "/images/people/ashley_main.jpg",
+  },
+  {
+    name: "Ashley Lee",
+    title: "Senior Research Software Engineer",
+    href: "/ashley-lee",
+    image: "/images/people/ashley_main.jpg",
+  },
+  {
+    name: "Ashley Lee",
+    title: "Senior Research Software Engineer",
+    href: "/ashley-lee",
+    image: "/images/people/ashley_main.jpg",
+  },
+  {
+    name: "Ashley Lee",
+    title: "Senior Research Software Engineer",
+    href: "/ashley-lee",
+    image: "/images/people/ashley_main.jpg",
+  },
+]
+
+export default async function AboutUs() {
+    return (
+      <div className="w-full">
+        <div className="relative w-full flex flex-col">
+          <div className="bg-purple-900">
+            <Hero image={"/images/hero-subroutes.jpeg"}>
+              <div className="relative flex-1 flex items-start w-full px-6 md:px-24 bg-gradient-to-t from-black/0 via-black/10 to-black/65 z-5">
+                <div className="absolute top-[12%] flex flex-col text-white space-y-6 w-[80vw]">
+                  <TextAnimate className="font-bold text-6xl md:text-8xl">
+                    About Us
+                  </TextAnimate>
+                  <p className="text-4xl font-semibold leading-[1.5]">
+                    The Center for Computation and Visualization provides high-performance computing and visualization services to the Brown community. We also collaborate with researchers on projects across vast range of disciplines.
+                  </p>
+                </div>
+              </div>
+            </Hero>
+          </div>
+        </div>
+
+        {/* Intro to OIT */}
+        <section className="content-wrapper py-24">
+          <SectionHeader title="Office of Information Technology" align="center" />
+          <Card className="w-full border-none shadow-none rounded-none">
+            <CardContent className="max-w-[1440px] mx-auto max-h-[600px] flex items-center px-6 py-10">
+              <p className="text-black text-xl">
+                The Center for Computation and Visualization (CCV) is a center within the University's central IT organization, which is the Office of Information Technology (OIT). In addition to building and maintaining the University's hundreds of enterprise software, systems, and hardware, OIT is also responsible for driving the technological progress that enables scientific research. Executing on the University's research mission is the key role that CCV plays in OIT.
```

Comment:
There are links for the following on the original page -- https://ccv.brown.edu/about
[Office of Information Technology (OIT)](https://it.brown.edu/) and [hundreds of enterprise software, systems, and hardware](https://it.brown.edu/services)

let's add those back in for now

Reply:
added !